compiler/rustc_infer/src/infer/error_reporting/nice_region_error/.different_lifetimes.rs.rustfmt

```diff
@@ -0,0 +1,174 @@
+//! Error Reporting for Anonymous Region Lifetime Errors
+//! where both the regions are anonymous.
```

Comment:
I don't think you meant to commit this file.

Reply:
oh so embarrasing!

Reply:
That's okay, just wanted to let you know :)